packages/web/pages/index.tsx

```diff
@@ -3,9 +3,13 @@ import Image from 'next/image';
 import { useEffect, useState } from 'react';
 
 const Home: NextPage = () => {
-  const [isMac, setIsMac] = useState(false);
+  const [os, setOs] = useState('windows');
   useEffect(() => {
-    setIsMac(navigator.platform.toUpperCase().indexOf('MAC') === 0);
+    if (navigator.platform.toUpperCase().indexOf('MAC') === 0) {
```

Comment:
do we need useState here? seems like this can be derived in the render() call without it -- could useMemo if the navigator access is slow somehow

Reply:
i remember the window object was causing problem during build when next tried running it in a server side bundle or something. Putting in useEffect was to make sure it only runs on the client side